docker/docker-compose.yml

```diff
@@ -37,4 +21,5 @@ services:
       - "9191:9191"
     volumes:
       - "./.env:/app/.env"
-      - "/data_local/application-data/itwewina/db.sqlite3:/app/CreeDictionary/db.sqlite3"
+      - "/data_local/application-data/itwewina/db/:/app/CreeDictionary/db/"
+      - "/data_local/application-data/itwewina/search-quality/:/app/CreeDictionary/search_quality/sample_results/"
```

Comment:
I have created the requisite directories and hard links!

```
easantos@altlab-itw:~$ ls -liRF /data_local/application-data/itwewina/
/data_local/application-data/itwewina/:
total 1107292
3034048 -rw-r--r-- 1 root     root            496 Apr  1 11:57 README
3034044 drwxr-xr-x 2 itwewina itwewina       4096 Apr  1 11:52 db/
3014661 -rw-r--r-- 2 itwewina itwewina 1133850624 Apr  1 10:19 db.sqlite3
3034045 drwxr-xr-x 2 itwewina itwewina       4096 Apr  1 11:51 search-quality/

/data_local/application-data/itwewina/db:
total 1107280
3014661 -rw-r--r-- 2 itwewina itwewina 1133850624 Apr  1 10:19 db.sqlite3

/data_local/application-data/itwewina/search-quality:
total 0
```

Reply:
Thank you!